source/tutorials/lamp-server.rst

```diff
@@ -1,4 +1,4 @@
-.. _web-server-install:
+.. _lamp-server-install:
```

Comment:
We'd like to preserver the URL. Doc title can change though.

Reply:
my reason for changing the URL is that "web-server-install" is too generic.  It could be Apache, nginx, etc.

Reply:
@krisdale.  Please take a quick look to confirm that this doc has low hits. 
Please submit a ticket. Thanks.

Reply:
Confirmed: hit rate is low. Keep as is, @bktan8.